src/ResourceManagement/CosmosDB/CassandraTableImpl.cs

```diff
@@ -0,0 +1,460 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Management.CosmosDB.Fluent
+{
+    using Microsoft.Azure.Management.CosmosDB.Fluent.Models;
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Linq;
+
+    using DefinitionParentT = CassandraKeyspace.Definition.IWithAttach<CosmosDBAccount.Definition.IWithCreate>;
+    using UpdateDefinitionParentT = CassandraKeyspace.Definition.IWithAttach<CosmosDBAccount.Update.IWithOptionals>;
+
+    internal partial class CassandraTableImpl :
+        ExternalChildResource<ICassandraTable,
+            CassandraTableGetResultsInner,
+            ICassandraKeyspace,
+            CassandraKeyspaceImpl>,
+        ICassandraTable,
+        CassandraTable.Definition.IDefinition<DefinitionParentT>,
+        CassandraTable.Definition.IDefinition<UpdateDefinitionParentT>,
+        CassandraTable.Definition.IDefinition<CassandraKeyspace.Update.IUpdate>,
+        CassandraTable.Update.IUpdate
+    {
+        private ICassandraResourcesOperations Client { get { return Parent.Parent.Manager.Inner.CassandraResources; } }
+        private CassandraTableCreateUpdateParameters createUpdateParameters;
+        private ThroughputSettingsUpdateParameters throughputSettingsToUpdate;
+
+        private string Location
+        {
+            get
+            {
+                return Parent.Parent.RegionName.ToLower();
+            }
+        }
+
+        private string ResourceGroupName
+        {
+            get
+            {
+                return Parent.Parent.ResourceGroupName;
+            }
+        }
+
+        private string AccountName
+        {
+            get
+            {
+                return Parent.Parent.Name;
+            }
+        }
+
+        private string CassandraKeyspaceName
+        {
+            get
+            {
+                return Parent.Name();
+            }
+        }
+
+        internal CassandraTableImpl(string name, CassandraKeyspaceImpl parent, CassandraTableGetResultsInner inner)
+            : base(name, parent, inner)
+        {
+            SetCreateUpdateParameters();
+        }
+
+        private void SetCreateUpdateParameters()
```

Comment:
For naming, it should be `initializeCreateUpdateParameters`

Normally, set would be property to set value from external.

Reply:
done